definitions/schema-products.graphqls

```diff
@@ -112,6 +90,13 @@ type ProductDetails {
   longDescription: String
   "For building canonical URL to PDP."
   slug: String!
+  "Details of category/classification for breadcrumbs."
```

Comment:
Cannot we use only `Product`?

Reply:
Honestly, I'm not sure. I don't know if is there a reason for having `ProductDetails` instead of just `Product`. Do you any context on this?

Reply:
I think it is the same situation as for `productsListing` and `productSummaryListing`. We can decide which one to use, FE uses `Product`.

Reply:
Ok, I'll do that.

Reply:
Fixed on [c241c06](https://github.com/Mindera/Alfie-Mocks/pull/2/commits/c241c066d196e1fb79bb7bbbf4ed7fbc8e3c49aa)